packages/cli/tests/unit/depends.spec.ts

```diff
@@ -20,6 +20,7 @@ describe('Depends', () => {
   beforeAll(async () => verbose(process.env.DEBUG === 'true'));
   beforeEach(async () => {
     appMapDir = tmp.dirSync().name;
+    expected = join(appMapDir, 'user_page_scenario').split('\\').join('/');
```

Comment:
`replaceAll` could work better :) also, if you just want to ensure usage of unix separators, you can use `path.posix.join`.

Reply:
For some reason, when I use `replaceAll` I get this error:
![image](https://user-images.githubusercontent.com/45714532/195427239-1323abbb-7cd3-4469-a767-1b87449fbc65.png)

I changed it to `replace`, though.